doc/packed_data.rdoc

```diff
@@ -327,14 +327,14 @@ for one byte in the input or output string.
 - <tt>'A'</tt> - Arbitrary binary string (space padded; count is width);
   +nil+ is treated as the empty string:
 
-    ['foo'].pack('A')    # => "f"
-    ['foo'].pack('A*')   # => "foo"
-    ['foo'].pack('A2')   # => "fo"
-    ['foo'].pack('A4')   # => "foo "
-    [nil].pack('A')      # => " "
-    [nil].pack('A*')     # => ""
-    [nil].pack('A2')     # => "  "
-    [nil].pack('A4')     # => "    "
+    ['foo'].pack('A')  # => "f"
+    ['foo'].pack('A*') # => "foo"
+    ['foo'].pack('A2') # => "fo"
+    ['foo'].pack('A4') # => "foo "
+    [nil].pack('A')    # => " "
+    [nil].pack('A*')   # => ""
+    [nil].pack('A2')   # => "  "
+    [nil].pack('A4')   # => "    "
```

Comment:
The previous doesn't seem inconsistent.
Moving comments to the longest line + 1 column?

Reply:
I apologize if I'm not understanding your question here. Yes, throughout most of this file, each sequence of code statements was commented to the longest line + 1 column, so I applied that rule here as well.

Reply:
Is there something I can do to move this PR forward?